python/lsst/ip/diffim/dipoleMeasurement.py

```diff
@@ -172,76 +206,14 @@ class DipoleMeasurementTask(SingleFrameMeasurementTask):
 
 \section ip_diffim_dipolemeas_Config       Configuration parameters
 
-See \ref DipoleMeasurementConfig and \ref DipoleClassificationConfig
+See \ref DipoleMeasurementConfig
 
 #-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
 
 \section ip_diffim_dipolemeas_Metadata   Quantities set in Metadata
 
 No specific values are set in the Task metadata.  However, the Source schema are modified to store the
-results of the dipole-specific measurements.  These additional fields include:
-
- . classification.dipole : probability of being a dipole
-
- . centroid.dipole.naive.pos : unweighted 3x3 first moment centroid
-
- . centroid.dipole.naive.pos.err : covariance matrix for centroid.dipole.naive.pos
-
- . centroid.dipole.naive.pos.flags : set if the centroid.dipole.naive.pos measurement did not fully succeed
-
- . centroid.dipole.naive.neg : unweighted 3x3 first moment centroid
-
- . centroid.dipole.naive.neg.err : covariance matrix for centroid.dipole.naive.neg
-
- . centroid.dipole.naive.neg.flags : set if the centroid.dipole.naive.neg measurement did not fully succeed
-
- . flux.dipole.naive.pos : raw flux counts
-
- . flux.dipole.naive.pos.err : uncertainty for flux.dipole.naive.pos
-
- . flux.dipole.naive.pos.flags : set if the flux.dipole.naive.pos measurement failed
-
- . flux.dipole.naive.neg : raw flux counts
-
- . flux.dipole.naive.neg.err : uncertainty for flux.dipole.naive.neg
-
- . flux.dipole.naive.neg.flags : set if the flux.dipole.naive.neg measurement failed
-
- . flux.dipole.naive.npos : number of positive pixels
-
- . flux.dipole.naive.nneg : number of negative pixels
-
- . flux.dipole.psf.pos : jointly fitted psf flux counts
-
- . flux.dipole.psf.pos.err : uncertainty for flux.dipole.psf.pos
-
- . flux.dipole.psf.pos.flags : set if the flux.dipole.psf.pos measurement failed
-
- . flux.dipole.psf.neg : jointly fitted psf flux counts
-
- . flux.dipole.psf.neg.err : uncertainty for flux.dipole.psf.neg
-
- . flux.dipole.psf.neg.flags : set if the flux.dipole.psf.neg measurement failed
-
- . flux.dipole.psf.chi2dof : chi2 per degree of freedom of fit
-
- . flux.dipole.psf.centroid : average of the postive and negative lobe positions
-
- . flux.dipole.psf.centroid.err : covariance matrix for flux.dipole.psf.centroid
-
- . flux.dipole.psf.centroid.flags : set if the flux.dipole.psf.centroid measurement did not fully succeed
-
- . flux.dipole.psf.neg.centroid : psf fitted center of negative lobe
-
- . flux.dipole.psf.neg.centroid.err : covariance matrix for flux.dipole.psf.neg.centroid
-
- . flux.dipole.psf.neg.centroid.flags : set if the flux.dipole.psf.neg.centroid measurement did not fully succeed
-
- . flux.dipole.psf.pos.centroid : psf fitted center of positive lobe
-
- . flux.dipole.psf.pos.centroid.err : covariance matrix for flux.dipole.psf.pos.centroid
-
- . flux.dipole.psf.pos.centroid.flags : set if the flux.dipole.psf.pos.centroid measurement did not fully succeed
```

Comment:
These are redundant with the docstrings on the original plugins.  Not a single developer who has updated the default plugins in DipoleMeasurementConfig in the last 2 years has bothered to copy and paste the plugin docstrings to this task documentation. I believe this will continue to be the rule and not the exception, and that this falls under code-duplication principles.